cli/command/image/build_buildkit.go

```diff
@@ -103,6 +103,11 @@ func runBuildBuildKit(dockerCli command.Cli, options buildOptions) error {
 		} else {
 			dockerfileDir = options.context
 		}
+
+		if err := build.OpenDockerignore(contextDir); err != nil {
```

Comment:
When using buildkit, a session is opened; wondering if an error should be produced on any file that is attempted to send to the daemon (instead of checking for specific files).

@tonistiigi wdyt?